template.yaml

```diff
@@ -205,6 +205,7 @@ Resources:
       Handler: no.unit.nva.cristin.person.handler.FetchCristinPersonHandler::handleRequest
       Runtime: java11
       MemorySize: 512
+      Role: !GetAtt ReadCristinAuthenticationRole.Arn
```

Comment:
Only fetch and not query for authenticated user?

Reply:
Authenticated user can do query, but does not get 'NationnalIdentificationNumber' or 'reserved' in query response